lib/src/upgrade_messages.dart

```diff
@@ -638,6 +638,8 @@ class UpgraderMessages {
       case 'fil':
       case 'fr':
       case 'de':
+        message = 'Versionshinweise';
```

Comment:
@LenhartStephan I think fr and others would fall through to de.

Reply:
I will take this message and copy it into the code myself. No need for another update or PR. Thanks for the translation.